src/cryptoadvance/specter/config.py

```diff
@@ -57,6 +57,7 @@ class BaseConfig(object):
     # The prefix for extensions which don't get access to the session cookie (if SPECTER_URL_PREFIX isn't compromised)
     ISOLATED_CLIENT_EXT_URL_PREFIX = "/ext"
 
+    HOST = os.getenv("HOST", "127.0.0.1")
```

Comment:
Can't we move that to the config of the extension?

Reply:
I need the `app.config["HOST"]` in the notification extension.  Adding it to the global BaseConfig, does make it consistent with `PORT`.  And if another `HOST` value if given for the server then the config value is updated here https://github.com/relativisticelectron/specter-desktop/blob/bf03c3236958c49405b7d56c71e0b6ab67c9f5e4/src/cryptoadvance/specter/cli/cli_server.py#L127

I don't really see how to move this to the service.